### PR TITLE
Contributing pBWA

### PR DESCRIPTION
**Pull request by tloizou**

### CODE REVIEW

File: easybuild/easyconfigs/p/pBWA/pBWA-0.5.9_1.21009-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# $Id$
+#
+##
+easyblock = "MakeCp"
+
+name = 'pBWA'
+version = '0.5.9_1.21009'
+
+homepage = 'http://pbwa.sourceforge.net/'
+description = """pBWA is a parallel implementation of the popular software BWA. It was developed by modifying the BWA source code with the OpenMPI C library on the SHARCNET. pBWA has been successfully tested on other systems with the most basic OpenMPI installs. pBWA currently implements three commands from BWA: aln, samse, and sampe. pBWA retains and improves upon the multithreading provided by BWA while adding efficient parallelization for the above listed functions. pBWA has shown that its wall-time speedup is bounded only by the size of the parallel system available as pBWA can run on any number of nodes and/or cores simultaneously."""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/pBWA/pBWA-0.5.9_1.21009-goolf-1.4.10.eb]: please wrap into multiple lines, for readability!

also, trim redundant whitespace visible below (ie. extra lines, make dependencies more compact)

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/p/pBWA/pBWA-0.5.9_1.21009-goolf-1.4.10.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/p/pBWA/pBWA-0.5.9_1.21009-goolf-1.4.10.eb]: style remarks fixed via https://github.com/tloizou/easybuild-easyconfigs/pull/2

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/603c7aa2b958e5b6fe3d for a full test report.

**Comment by boegel** [comment]
Going in, thanks @tloizou!

**Pull request merged**